salt/metalk8s/orchestrate/upgrade/init.sls

```diff
@@ -53,9 +62,16 @@ Skip node {{ node }}, already in {{ node_version }} newer than {{ dest_version }
 
   {%- else %}
 
+Install apiserver-proxy on {{ node }}:
+  salt.state:
+    - tgt: {{ node }}
+    - sls:
+      - metalk8s.kubernetes.apiserver-proxy
+    - saltenv: {{ saltenv }}
```

Comment:
This one should require `salt: Ensure all apiservers serve a certificate for 127.0.0.1`

Reply:
No, there's no dependency on that. `kubelet` reconfiguration requires the proxy to be up, and the cert to be resigned. But the proxy can happily be deployed and run, it doesn't really care about the services it's proxying to (at the TCP level).

Reply:
I added a `require` on `salt: Ensure all apiservers serve a certificate for 127.0.0.1` to `Deploy node {{ node }}` instead.